docs/automation/cucumber-expressions.md

```diff
@@ -22,15 +22,21 @@ To match for a simple text, just use the text as cucumber expression.
 
 ### Parameters
 
-Parameters can be defined using the `{parameter-type}` syntax. Where `parameter-type` can be any of the following:
+Parameters can be defined using the `{parameter-type}` syntax. The typing name is case sensitive. Where `parameter-type` can be any of the following:
 
-* A short name for some simple built-in types: `{int}`, `{long}`, `{byte}`, `{float}`, `{double}`, `{decimal}`
+* A short name for some simple built-in types: `{int}`, `{long}`, `{byte}`, `{float}`, `{double}`, `{decimal}`, `{DateTime}`
```

Comment:
The `{DateTime}` is an example of bullet point 5 ("A type name without namespace that is supported by Reqnroll as a parameter type"), so it should be rather listed there. (Maybe bullet point 5 should be slit to built-in and user defined types.)